packages/loopring_v3/contracts/impl/ExchangeV3.sol

```diff
@@ -625,12 +442,48 @@ contract ExchangeV3 is IExchangeV3
         view
         returns (uint)
     {
-        uint24 accountID = (owner == address(0)) ? 0 : state.getAccountID(owner);
         uint16 tokenID = state.getTokenID(token);
-        return state.amountWithdrawable[accountID][tokenID];
+        return state.amountWithdrawable[owner][tokenID];
+    }
+
+    function notifyForcedRequestTooOld(
+        uint24 accountID,
+        address token
+        )
+        external
+        override
+        nonReentrant
+    {
+        uint16 tokenID = state.getTokenID(token);
+        ExchangeData.ForcedWithdrawal storage withdrawal = state.pendingForcedWithdrawals[accountID][tokenID];
+        require(withdrawal.timestamp != 0, "WITHDRAWAL_NOT_TOO_OLD");
+
+        // Check if the withdrawal has indeed exceeded the time limit
+        require(now >= withdrawal.timestamp + ExchangeData.MAX_AGE_FORCED_REQUEST_UNTIL_WITHDRAW_MODE(), "WITHDRAWAL_NOT_TOO_OLD");
+
+        // Enter withdrawal mode
+        state.withdrawalModeStartTime = now;
+
+        emit WithdrawalModeActivated(state.withdrawalModeStartTime);
     }
 
     // -- Agents --
+    function whitelistAgents(
+        address[] calldata agents,
+        bool[]    calldata whitelisted
+        )
+        external
+        override
+        nonReentrant
+        onlyOwner
+    {
+        require(agents.length == whitelisted.length, "INVALID_DATA");
```

Comment:
delay taking effect if `whitelisted == true`

Reply:
Brecht, I think this can also be achieved through the delayed owner contract, right?

Reply:
Yep!